Controller/DashboardAdminController.php

```diff
@@ -56,7 +58,7 @@ public function composeAction(Request $request = null)
 
         $csrfProvider = $this->get('form.csrf_provider');
 
-        return $this->render('SonataDashboardBundle:DashboardAdmin:compose.html.twig', array(
+        return $this->render($template, array(
```

Comment:
Write `return $this->render($this->admin->getTemplate('compose'), array` instead